libs/blocks/quiz-results/quiz-results.css

```diff
@@ -7,3 +7,59 @@
 .quiz-results.show {
   visibility: unset;
 }
+
+.section.phone-only,
+.section.tablet-portrait-up,.section.tablet-landscape-up, 
+.section.desktop-up,.section.big-desktop-up {
+  display: none;
+}
+
+@media screen and (max-width: 599px) { 
+  .section.section.phone-only {
+    display: block;
```

Comment:
We have a mobile first approach, meaning that default (mobile) properties can be written without a media query.

Reply:
in this case we need to display some titles only on mobile or switch content for mobile view.
this is why we are working with section metadata do add some phone-only classes.

Style side we are creating the quiz mobile first

cc: @echampio-at-adobe  @fullcolorcoder @colloyd

Reply:
@elaineskpt I think this is mostly about readability, and keeping it clean for future maintenance. It's always cleaner to see CSS inside of a media query and know that this is always non-mobile style. Max-width could save a few lines for now maybe, but we could say .phone-only is set to display: block, then set to none in the min-width:600 section to keep in alignment with the mobile first coding style. 

Bootstrap does a similar approach https://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements

Reply:
Just added an update for it @fullcolorcoder @narcis-radu